ftest.sh

```diff
@@ -154,6 +154,10 @@ sudo bash -c \"set -ex
 if [ \\\"\\\$(ulimit -c)\\\" != \\\"unlimited\\\" ]; then
     echo \\\"*  soft  core  unlimited\\\" >> /etc/security/limits.conf
 fi
+if [ \\\"\\\$(ulimit -l)\\\" != \\\"unlimited\\\" ]; then
+    echo \\\"*  soft  memlock  unlimited\\\" >> /etc/security/limits.conf
+    echo \\\"*  hard  memlock  unlimited\\\" >> /etc/security/limits.conf
```

Comment:
You don't need to set a **hard** limit if the **soft** limit is `unlimited`.

Reply:
Done.

Reply:
FYI, seems the **hard** limit is back.  It's not going to cause any particular negative effects, but is just superfluous.

That said, this seems to be working now:

```
...
[2020-06-01T22:07:30.124Z]       *  soft  core  unlimited
[2020-06-01T22:07:30.124Z]       *  hard  memlock  unlimited
[2020-06-01T22:07:30.124Z]       *  soft  memlock  unlimited
[2020-06-01T22:07:30.124Z] Getting ulimit -a on ['wolf-138', 'wolf-139']
[2020-06-01T22:07:30.124Z] Running on wolf-[138-139]: ulimit -a
[2020-06-01T22:07:30.383Z]     wolf-[138-139]: rc=0, output: 
...
[2020-06-01T22:07:30.383Z]       max locked memory       (kbytes, -l) unlimited
```

Don't forget to consider how this is effected in a production environment.  That means there probably needs to be a ticket open to either document the need for this setting, or to have it set automatically by the control plane, etc.

Probably us having to set this in `ftest.sh` is a bug as it means that in a production environment, where obviously `ftest.sh` is not run, there is a configuration _gap_.